### PR TITLE
added composite components that will be used in UI rendering

### PR DESCRIPTION
**Pull request by GuptaEesh**
Added components that will be used as it is in ecomm-pages.

>Footer
Loader
Navbar
Product-cards
filter-holder
Language drop

### CODE REVIEW

**Review by shivampandey0 — APPROVED** [review]
Code looks good and well structured with atomic-level components.
Good to merge

File: src/components/composite/Product-cards/price-banner.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                                local_offer
+                            </span>{' '}
+                            Coupons
+                        </>
+                    }
+                />
+            </header>
+            <h2>Price Details:</h2>
+            <section>
+                <table>
```
Comment (lannisterrr) [inline src/components/composite/Product-cards/price-banner.js:34]: table can be a seperate component.

**Review by lannisterrr — APPROVED** [review]
Good use of custom hooks  and very structured code. Made some suggestions.

File: src/components/composite/Product-cards/price-banner.js (hunk 1 above)
Comment (GuptaEesh) [reply src/components/composite/Product-cards/price-banner.js:34]: Components should be reusable.
1. It is not used elsewhere.
2. It is not blocking the view as it's a behind-the-scenes component.

**Comment by GuptaEesh** [comment]
Got 2 reviews. Merging the branch now.

**Pull request merged**